Gemfile

```diff
@@ -2,47 +2,46 @@
 
 source 'https://rubygems.org'
 
-gem 'rails'
-# Lets us easily inline our svg to allow styling. Supports the rails asset pipeline.
-gem 'bootsnap'
-gem 'inline_svg'
+group :default do
+  gem 'bootsnap'
+  gem 'rails'
+  # Lets us easily inline our svg to allow styling. Supports the rails asset pipeline.
+  gem 'inline_svg'
 
-gem 'exception_notification'
-gem 'rake'
-gem 'state_machines'
-gem 'webpacker'
+  gem 'exception_notification'
+  gem 'rake'
+  gem 'state_machines'
+  gem 'webpacker'
 
-# Used in the setting object, allows access by object and hash notation.
-gem 'hashie'
+  # Used in the setting object, allows access by object and hash notation.
+  gem 'hashie'
 
-# Communications with JSON APIs, allows us to begin migration to the new Sequencescape API
-gem 'json_api_client', github: 'sanger/json_api_client', branch: 'merge_upstream'
+  # Communications with JSON APIs, allows us to begin migration to the new Sequencescape API
+  gem 'json_api_client', github: 'sanger/json_api_client', branch: 'merge_upstream'
 
-# Older Sequencescape API
-gem 'sequencescape-client-api', '~> 0.5.0', require: 'sequencescape'
-# Speed up json encoding/decoding with oj
-gem 'oj'
+  # Older Sequencescape API
+  gem 'sequencescape-client-api', '~> 0.5.0', require: 'sequencescape'
+  # Speed up json encoding/decoding with oj
+  gem 'oj'
 
-gem 'pmb-client', '0.1.0', github: 'sanger/pmb-client'
-gem 'sprint_client'
-
-gem 'puma'
-gem 'sanger_barcode_format', github: 'sanger/sanger_barcode_format', branch: 'development'
+  gem 'pmb-client', '0.1.0', github: 'sanger/pmb-client'
+  gem 'sprint_client'
 
+  gem 'puma'
+  gem 'sanger_barcode_format', github: 'sanger/sanger_barcode_format', branch: 'development'
```

Comment:
weird that this is development not master

Reply:
Yeah, it was probably a case of me wanted to run with some improvements, but they hadn't yet been merged to master. We should probably just add it to ruby-gems instead,